lib/elixir/lib/code/formatter.ex

```diff
@@ -1017,7 +1017,7 @@ defmodule Code.Formatter do
     {target_doc, state} = quoted_to_algebra(target, :no_parens_arg, state)
 
     {{call_doc, state}, wrap_in_parens?} =
-      call_args_to_algebra(args, meta, context, :required, true, state)
+      call_args_to_algebra(args, meta, context, :skip_if_empty_do_end, true, state)
```

Comment:
I was thinking that **the other use** of `:required` in the code base would become `:skip_if_empty_do_end`. That's because what we are doing exactly in here is not skip_if_empty_do_end. We are adding if empty do end.

Reply:
You're right. I would suggest naming it `:skip_if_only_do_end`, since the do-end is the only argument. `:skip_if_empty_do_end` to me sounds like it requires the do-end block to be empty, which would make no sense.

Reply:
Sounds good!